ckanext/stadtzhtheme/dcat/profiles.py

```diff
@@ -0,0 +1,272 @@
+from ckanext.dcat.profiles import RDFProfile
+from ckanext.dcat.utils import resource_uri, publisher_uri_from_dataset_dict
+
+import rdflib
+from rdflib import URIRef, BNode, Literal
+from rdflib.namespace import Namespace, RDF, XSD, SKOS, RDFS
+
+import pylons
+import logging
+log = logging.getLogger(__name__)
+
+
+
+DCT = Namespace("http://purl.org/dc/terms/")
+DCAT = Namespace("http://www.w3.org/ns/dcat#")
+ADMS = Namespace("http://www.w3.org/ns/adms#")
+VCARD = Namespace("http://www.w3.org/2006/vcard/ns#")
+FOAF = Namespace("http://xmlns.com/foaf/0.1/")
+SCHEMA = Namespace('http://schema.org/')
+TIME = Namespace('http://www.w3.org/2006/time')
+LOCN = Namespace('http://www.w3.org/ns/locn#')
+GSP = Namespace('http://www.opengis.net/ont/geosparql#')
+OWL = Namespace('http://www.w3.org/2002/07/owl#')
+SPDX = Namespace('http://spdx.org/rdf/terms#')
+XML = Namespace('http://www.w3.org/2001/XMLSchema')
+
+GEOJSON_IMT = 'https://www.iana.org/assignments/media-types/application/vnd.geo+json'
+
+namespaces = {
+    'dct': DCT,
+    'dcat': DCAT,
+    'adms': ADMS,
+    'vcard': VCARD,
+    'foaf': FOAF,
+    'schema': SCHEMA,
+    'time': TIME,
+    'skos': SKOS,
+    'locn': LOCN,
+    'gsp': GSP,
+    'owl': OWL,
+    'xml': XML,
+}
+
+ogd_theme_base_url = 'http://opendata.swiss/themes/'
+mapping_groups_dict = {
+    'arbeit-und-erwerb': ['work'],
+    'finanzen': ['finances'],
+    'preise': ['prices'],
+    'tourismus': ['tourism'],
+    'volkswirtschaft': ['national-economy'],
+    'wirtschaft': ['national-economy'],
+    'verwaltung': ['administration'],
+    'soziales': ['social-security'],
+    'freizeit': ['territory'],
+    'umwelt': ['territory'],
+    'politik': ['politics'],
+    'mobilitat': ['mobility'],
+    'kultur': ['culture'],
+    'kriminalitat': ['crime', 'public-order'],
+    'gesundheit': ['health'],
+    'basiskarten': ['territory', 'geography'],
+    'energie': ['energy'],
+    'bildung': ['education'],
+    'bevolkerung': ['population'],
+    'bauen-und-wohnen': ['construction'],
+}
+mapping_rights_dict = {
+    'cc-by-sa': 'NonCommercialAllow - CommercialAllow - ReferenceRequired',
+    'cc-zero': 'NonCommercialAllow - CommercialAllow - ReferenceNotRequired',
+}
+
+# Values 'sporadisch oder unregelmaessig' and 'keines'
+# do not have an equivalent in OGD and are ignored
+mapping_accrualPerdiodicty = {
+    'halbjaehrlich': 'semiannual',
+    'jaehrlich': 'annual',
+    'laufend': 'continuous',
+    'monatlich': 'monthly',
+    'quartalsweise': 'quarterly',
+    'stuendlich': 'continuous',
+    'taeglich': 'daily',
+    'woechentlich': 'weekly',
+}
```

Comment:
Use to values for dct:accrualPeriodicity from http://handbook.opendata.swiss/en/library/ch-dcat-ap, maybe even use `http://purl.org/cld/freq/completelyIrregular` if needed